sample/src/main/kotlin/com/slack/circuit/sample/home/BottomNavItem.kt

```diff
@@ -21,14 +21,18 @@ import androidx.compose.material.icons.filled.Info
 import androidx.compose.ui.graphics.vector.ImageVector
 import com.slack.circuit.Screen
 import com.slack.circuit.sample.petlist.AboutScreen
+import com.slack.circuit.sample.petlist.Filters
 import com.slack.circuit.sample.petlist.PetListScreen
 
-private const val DOGS_SCREEN_NAME = "Dogs"
+private const val DOGS_SCREEN_NAME = "Adoptables"
 private const val ABOUT_SCREEN_NAME = "About"
-const val DOGS_SCREEN_INDEX = 0
-const val ABOUT_SCREEN_INDEX = 1
```

Comment:
Just curious, why remove the named indexes in favor of magic numbers?

Reply:
just one magic number, and it's not really magic. Starting index 0 seems pretty standard to me 👀